pyaedt/siwave.py

```diff
@@ -46,7 +46,6 @@ class Siwave:
     """
 
     @property
-    @aedt_exception_handler
```

Comment:
@maxcapodi78 Is it expected to remove the decorator `@aedt_exception_handler` for siwave?

Reply:
I've removed from some properties where it's not needed